### PR TITLE
Updated benji

### PR DESCRIPTION
**Pull request by rishabh0098**
Added feature for finding files in pc.

### CODE REVIEW

**Comment by the-ethan-hunt** [comment]
An Error window is coming. Please solve the bug. It's a really nice feature :+1:

**Comment by rishabh0098** [comment]
But it's working here
Did you give path where to look for the file?

**Comment by the-ethan-hunt** [comment]
Can you please mail me the snapshots of the feature working via mail? :smile:

**Comment by rishabh0098** [comment]
I think its working for one file only.

**Comment by rishabh0098** [comment]
I've fixed the bug

File: benji.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def __init__(self):
      
         def OnEnter(self,event):
             put=self.txt.GetValue()
-            put=put.lower()
+            #put=put.lower()
```
Comment (BrianMwangi21) [inline benji.py]: the string is converted to lower for the if else conditions. commenting out the line means 'SEARCH BATMAN' won't work like 'search batman' when it should

**Comment by the-ethan-hunt** [comment]
@rishabh0098 , please solve the issue mentioned by @BrianMwangi21 so that I can merge it

File: benji.py (hunk 1 above)
Comment (rishabh0098) [reply benji.py]: but when searching for a file, filename is case sensitive, right?

Comment (BrianMwangi21) [reply benji.py]: Depends on the method you are using. But a simple search on my file explorer finds the file in whichever case, lower or upper

**Comment by rishabh0098** [comment]
solved the lower case issue

**Comment by rishabh0098** [comment]
@BrianMwangi21 Thanks for pointing out the issue

**Comment by BrianMwangi21** [comment]
@rishabh0098 you're welcome

**Comment by rishabh0098** [comment]
@the-ethan-hunt I've solved the issue you can now merge it

**Comment by rishabh0098** [comment]
I'm closing this PR, will start a new one

**Pull request closed without merging**